tests/drum/test_fit.py

```diff
@@ -136,16 +158,6 @@ def test_fit(
         weights,
         tmp_path,
     ):
-        # exceptions
-        if docker and framework not in [SKLEARN_BINARY, SKLEARN_REGRESSION]:
-            return
-        if (framework, problem) in [
-            (SKLEARN_BINARY, REGRESSION),
-            (SKLEARN_BINARY, ANOMALY),
-            (SKLEARN_REGRESSION, BINARY_TEXT),
-        ]:
-            return
-
         if framework == RDS:
             language = R_FIT
         else:
```

Comment:
I can't select it, but down below theres some other code that skips certain subsets of the test cases that you can remove now.